korman/exporter/animation.py

```diff
@@ -73,37 +73,55 @@ def fetch_animation_data(id_data):
         # to be listed on the SceneObject in a specific order. D'OH! So, always use these funcs.
         agmod, agmaster = self.get_anigraph_objects(bo, so)
         atcanim = self._mgr.find_create_object(plATCAnim, so=so)
+        globalanim = self._mgr.find_create_object(plAgeGlobalAnim, so=so)
```

Comment:
Because a plAgeGlobalAnim is a special type of plATCAnim and has all the same functions available, you might be able to simplify this and avoid creating two objects and a bunch of these `if` conditions.

Something like this might work:
```python
if anim_data.obj_sdl_anim:
  atcanim = self._mgr.find_create_object(plAgeGlobalAnim, so=so)
else:
  atcanim = self._mgr.find_create_object(plATCAnim, so=so)
```

Then you can mostly use `atcanim` without caring whether it's a plATCAnim or a plAgeGlobalAnim.

Reply:
I've reworked that section of the script a bit to try what you said and pushed the changes. It's now only creating a plAgeGlobalAnim for anything with an SDL state, but for some reason, it's still creating a supplementary ATCAnim object...likely due to the set_anim_params def I added?

Reply:
It's possible the plATCAnim is getting created by another modifier. There are 2 spots in properties/modifiers/anim.py that look like they create ATCAnims.

I can't see where you're actually calling the `set_anim_params` function...?

Reply:
I've taken out the `set_anim_params` def, but it's still creating an ATCAnim for the global animations. Is it possible it's giving the object a second pass and adding it then? Might explain the "write data files" lagging? Might also have to do with this part:

```
            # Fixme? Not sure if we really need to expose this...
            atcanim.easeInMin = 1.0
            atcanim.easeInMax = 1.0
            atcanim.easeInLength = 1.0
            atcanim.easeOutMin = 1.0
            atcanim.easeOutMax = 1.0
            atcanim.easeOutLength = 1.0
```